### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by derpachu** [derpachu/MI-449-SS17-740-css-intro-iMnt09#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://derpachu.github.io/MI-449-SS17-740-css-intro-iMnt09/) and meets the following criteria:

- [x] Your project is built off [this base HTML and CSS](https://gist.github.com/chrisvfritz/279c36ab6e2301e8a35526be028d8a32) (you may modify both pages however you want, e.g. adding classes or more elements)
- [x] The website looks (nearly) identical to [this mockup](https://i.imgur.com/pGemyxp.png) in the Chrome web browser -- colors must be exact, spacing/sizes should be within about `5px` (use the _Page Ruler_ extension to measure exact sizes/spacing; you might need to open the image in a new tab)
- [x] Any CSS selectors or properties may be used, though the project is possible using only what you saw in the lesson
- [x] No spacing HTML (e.g. `<br>`, `&amp;nbsp;`) is added to `index.html`
- [x] Add at least one link (`a` element) to the page and style them to be `purple` normally, but turn `orange` on hover

Am not sure if the font was correct. Assumed it was the same as the list provided

**Comment by chrisvfritz** [comment]
I haven't looked at the code yet, but the colors are looking fantastic! 🚀  For the sizes though, I'm seeing some noticeable differences from the mockup. You can see a side-by-side comparison below, with the mockup on the right and your project on the left:

![Mockup vs real code](https://i.imgur.com/7CJ2aDw.png)

As you can see:

- All of your text is much smaller
- The images are much smaller
- The orange line is quite a bit thicker
- There's quite a bit of more padding on the `header` and `section` elements

A few tips that may help:

- In your browser, remember to make sure you're completely zoomed in on the image
- Don't forget to use the _Page Ruler_ extension to measure exact pixel sizes of things - this will be much easier than guessing!

If you're stuck and need some more help, just let me know. 🙂  This kind of design work can be tricky at first!

**Comment by derpachu** [comment]
made some more adjustments but unsure on how to get good comparisons without some sort of distortion. Just copied all the measurements with the ruler but it still looks just a tiny bit off.

**Comment by chrisvfritz** [comment]
It's definitely looking _much_ better! 👍  There are still some significant differences though. It can be difficult to tell what might be happening without seeing it, so I would get the attention of a mentor today in class, then walk them through the process you're using.

To give some examples of what I'm seeing, your version's `max-width` (top) is much smaller than the mockup's (bottom):

![Mockup comparison](https://i.imgur.com/C4fsAtT.png)

Looking at your `h2` elements, yours (left) are also quite a bit smaller than the mockup (right) and you have a link around East Lansing. There's also noticeably less space between this orange line and the images:

![Mockup comparison](https://i.imgur.com/oRdB4ao.png)

**Comment by derpachu** [comment]
Made the adjustments.

**Comment by chrisvfritz** [comment]
The `max-width` looks a little closer, but it's still not quite right. Did you try putting the mockup in a separate window and putting them side-by-side, like in my screenshot? If you do that, you'll be able to see with 100% certainty if you have it.

The spacing around the line is actually a bit worse now - I think you may have misread me. In your version, there's actually "__less__ space between this orange line and the __images__" (I bolded the parts I think you misunderstood).

Remember, if you flag down a mentor in class, they'll be able to help you figure out what's going wrong and give you strategies to make this work easier. 🙂

**Comment by derpachu** [comment]
Corrected the spacing between images and lines. Not sure about the max width because on my browser and in the ruler it all matches up. I just used the photograph and just used proportions to try to get it to match on your end.

**Comment by chrisvfritz** [comment]
> Not sure about the max width because on my browser and in the ruler it all matches up. I just used the photograph and just used proportions to try to get it to match on your end.

Yeah, that's a problem I want to make sure we fix before moving on. The pixels should show up _exactly_ the same on every computer, so if it's showing up differently for you, then something is definitely going wrong that will cause the same problems in future projects. I know the class is over now, so if you still haven't talked with a mentor, reach out to me on Slack and we can schedule a time to troubleshoot over a video call.

**Comment by derpachu** [comment]
I think I finally found the issue with the image distortion and made changes

**Comment by chrisvfritz** [comment]
_Excellent!_ 🎉 :shipit: 🎈 🍰  Nice work iterating on this.
